### PR TITLE
Feature/ted 896

### PR DESCRIPTION
**Pull request by kaleanych**

### CODE REVIEW

File: dags/operators/DagBatchPipelineOperator.py [hunk 1]
```diff
@@ -19,7 +19,8 @@
 DEFAULT_NUBER_OF_CELERY_WORKERS = 144
 NOTICE_PROCESS_WORKFLOW_DAG_NAME = "notice_process_workflow"
 DEFAULT_START_WITH_TASK_ID = "notice_normalisation_pipeline"
-DEFAULT_PIPELINE_NAME_FOR_LOGS  = "unknown_pipeline_name"
+DEFAULT_PIPELINE_NAME_FOR_LOGS = "unknown_pipeline_name"
```
Comment (costezki) [inline dags/operators/DagBatchPipelineOperator.py:22]: what is this for?

**Review by costezki — COMMENTED** [review]
I thought that the validation bug shall touch on fewer files than this. 
Probably there are multiple issues implemented by this PR

File: dags/notice_validation_workflow.py [hunk 2]
```diff
@@ -33,14 +33,16 @@ def notice_daily_validation_workflow():
         ))
     )
     def validate_fetched_notices():
```
Comment (CaptainOfHacks) [inline dags/notice_validation_workflow.py:35]: write documentation

File: ted_sws/supra_notice_manager/services/supra_notice_validator.py [hunk 3]
```diff
@@ -68,3 +70,33 @@ def summary_validation_for_daily_supra_notice(notice_publication_day: day_type,
     # no notice_ids needed to be stored for supra_notice
```
Comment (CaptainOfHacks) [inline ted_sws/supra_notice_manager/services/supra_notice_validator.py:70]: why we keep comments in code?

**Review by CaptainOfHacks — APPROVED** [review]
Merge main into current branch, before merge into main.

File: dags/operators/DagBatchPipelineOperator.py (hunk 1 above)
Comment (kaleanych) [reply dags/operators/DagBatchPipelineOperator.py:22]: this is merged from main. ask Stefan

File: ted_sws/supra_notice_manager/services/supra_notice_validator.py (hunk 3 above)
Comment (kaleanych) [reply ted_sws/supra_notice_manager/services/supra_notice_validator.py:70]: as a reminder of what makes double sense (was requested to store notice_ids in validation report, but they are already in main object). code is commented to be reactivated. will remove it later if no complains will be reported

**Pull request merged**